package.json

```diff
@@ -1,210 +1,214 @@
-		"@babel/plugin-proposal-export-namespace-from": "^7.5.2",
-		"@babel/plugin-proposal-object-rest-spread": "^7.6.2",
-		"@babel/plugin-proposal-throw-expressions": "^7.2.0",
-		"@babel/plugin-syntax-dynamic-import": "^7.2.0",
-		"@babel/preset-env": "^7.6.3",
-		"@babel/preset-react": "^7.6.3",
-		"@babel/preset-typescript": "^7.6.0",
-		"@types/chart.js": "^2.8.7",
-		"@types/connect-mongo": "0.0.44",
-		"@types/dotenv": "^6.1.1",
-		"@types/escape-html": "0.0.20",
-		"@types/express": "^4.17.1",
-		"@types/express-session": "^1.15.14",
-		"@types/faker": "^4.1.6",
-		"@types/graphql": "^14.5.0",
-		"@types/jest": "^24.0.18",
-		"@types/mongodb": "^3.3.6",
-		"@types/node": "^12.7.12",
-		"@types/passport-azure-ad": "^4.0.3",
-		"@types/passport-github2": "^1.2.4",
-		"@types/passport-google-oauth20": "^2.0.2",
-		"@types/pino": "^5.8.11",
-		"@types/react": "16.9.5",
-		"@types/react-csv": "^1.1.1",
-		"@types/react-dom": "^16.9.1",
-		"@types/react-router": "^5.1.1",
-		"@types/react-router-dom": "^5.1.0",
-		"@types/react-select": "^3.0.5",
-		"@types/react-test-renderer": "^16.9.0",
-		"@types/react-virtualized": "^9.21.4",
-		"@types/styled-components": "4.1.8",
-		"@types/supertest": "^2.0.8",
-		"@types/yup": "^0.26.24",
-		"@typescript-eslint/eslint-plugin": "^2.3.3",
-		"@typescript-eslint/parser": "^2.3.3",
-		"babel-jest": "^24.9.0",
-		"babel-loader": "^8.0.6",
-		"concurrently": "^5.0.0",
-		"copy-webpack-plugin": "^5.0.4",
-		"css-loader": "^3.2.0",
-		"error-overlay-webpack-plugin": "^0.4.1",
-		"eslint": "^6.5.1",
-		"eslint-config-airbnb": "^18.0.1",
-		"eslint-config-prettier": "^6.4.0",
-		"eslint-import-resolver-typescript": "^2.0.0",
-		"eslint-import-resolver-webpack": "^0.11.1",
-		"eslint-plugin-import": "^2.18.2",
-		"eslint-plugin-jest": "^22.17.0",
-		"eslint-plugin-jsx-a11y": "^6.2.3",
-		"eslint-plugin-prettier": "^3.1.1",
-		"eslint-plugin-promise": "^4.2.1",
-		"eslint-plugin-react": "^7.16.0",
-		"eslint-plugin-react-hooks": "^2.1.2",
-		"eslint-plugin-typescript": "^0.14.0",
-		"eslint_d": "^8.0.0",
-		"file-loader": "^4.2.0",
-		"fork-ts-checker-webpack-plugin": "^1.5.0",
-		"friendly-errors-webpack-plugin": "^1.7.0",
-		"html-webpack-plugin": "^3.2.0",
-		"jest": "^24.9.0",
-		"jest-transform-css": "^2.0.0",
-		"mongodb-memory-server": "^5.2.8",
-		"nodemon": "^1.19.3",
-		"pino-pretty": "^3.2.2",
-		"prettier": "^1.18.2",
-		"react-test-renderer": "^16.10.2",
-		"style-loader": "^1.0.0",
-		"supertest": "^4.0.2",
-		"svg-inline-loader": "^0.8.0",
-		"thread-loader": "^2.1.3",
-		"ts-jest": "^24.1.0",
-		"ts-node-dev": "^1.0.0-pre.40",
-		"typedoc": "^0.15.0",
-		"typescript": "^3.6.4",
-		"webpack": "^4.41.0",
-		"webpack-dev-server": "^3.8.2",
-		"webpack-merge": "^4.2.2"
-	},
-	"browserslist": [
-		"last 2 chrome versions",
-		"last 2 firefox versions",
-		"last 2 safari versions",
-		"last 2 edge versions",
-		"last 2 ios versions",
-		"last 2 chromeandroid versions"
-	]
+  "name": "vaken",
```

Comment:
why would u reindent this entire file

Reply:
it was passing lint before

Reply:
some VScode lint artifact, changed back to old indentation